src/Components/Publishing/Sections/ArtworkCaption.tsx

```diff
@@ -189,27 +212,34 @@ export class ArtworkCaption extends React.Component<ArtworkCaptionProps, null> {
       return (
         <StyledClassicCaption layout={layout} className="display-artwork__caption">
           <Truncator>
-            <strong>
+            <ArtistName>
               {this.renderArtists()}
-            </strong>
+            </ArtistName>
 
             {this.renderTitleDate()}
+            {". "}
             {this.renderPartner()}
           </Truncator>
         </StyledClassicCaption>
       )
 
-      // Other
+      // Default (Standard + Feature)
     } else {
       return (
         <StyledArtworkCaption layout={layout} sectionLayout={sectionLayout} className="display-artwork__caption">
-          <Truncator>
-            <span className="artist-name">
-              {this.renderArtists()}
-            </span>
+          <ArtistName>
+            {this.renderArtists()}
+          </ArtistName>
 
-            {this.renderTitleDatePartner()}
-          </Truncator>
+          <div>
+            <Truncator>
```

Comment:
Seems like Truncator could be broken out into its own component.

Reply:
Mmm I think it is?

Reply:
Own file rather